library/src/main/java/com/codemonkeylabs/fpslibrary/FrameDataCallback.java

```diff
@@ -0,0 +1,17 @@
+package com.codemonkeylabs.fpslibrary;
+
+/**
+ * Created by brianplummer on 11/12/15.
+ */
+public interface FrameDataCallback
+{
+    /**
+     * this is called for every doFrame() on the choreographer callback
+     * use this very judiciously.  you don't even want to Log.d() from here, bad idea
```

Comment:
Logging synchronously from here is bad idea as doFrame will be called every 16-32ms.